### PR TITLE
Update layout properites of KCBasic loan card to distribute content o…

### PR DESCRIPTION
**Pull request by mcstover**
…ver the full height of the carousel container.

Latest look:
image

### CODE REVIEW

File: src/components/LoanCards/KivaClassicBasicLoanCard.vue [hunk 1]
```diff
@@ -65,7 +65,9 @@
 		<borrower-name
 			v-if="!isLoading"
 			class="tw-mb-1 tw-text-h3"
+			:max-length="50"
 			:name="borrowerName"
+			style="min-height: 60px;"
```
Comment (mcstover) [inline src/components/LoanCards/KivaClassicBasicLoanCard.vue:70]: Text h3 has a line-height of 30 so we double that to get a baseline height. min-height allows the container to scale if someone zooms their screen...

File: src/components/LoanCards/KivaClassicBasicLoanCard.vue [hunk 2]
```diff
@@ -93,12 +95,12 @@
...
 		<kv-loading-paragraph
 			v-if="isLoading"
-			class="tw-mb-1.5" :style="{width: '100%', height: '5.5rem'}"
+			class="tw-mb-1.5 tw-flex-grow" :style="{width: '100%', height: '5.5rem'}"
 		/>
 
 		<loan-use
 			v-if="!isLoading"
-			class="tw-mb-2.5"
+			class="tw-mb-2.5 tw-flex-grow"
```
Comment (mcstover) [inline src/components/LoanCards/KivaClassicBasicLoanCard.vue:103]: This item is "growing" to take up extra available space with the result of pushing the following items to the bottom.

File: src/components/LoanCards/KivaClassicBasicLoanCard.vue [hunk 3]
```diff
@@ -128,12 +130,12 @@
...
 		<kv-loading-placeholder
 			v-if="isLoading"
-			class="tw-rounded" :style="{width: '9rem', height: '3rem'}"
+			class="tw-rounded tw-self-start" :style="{width: '9rem', height: '3rem'}"
 		/>
 
 		<kv-button
 			v-if="!isLoading && !allSharesReserved"
-			class="tw-mb-2"
+			class="tw-mb-2 tw-self-start"
```
Comment (mcstover) [inline src/components/LoanCards/KivaClassicBasicLoanCard.vue:138]: b/c the button would go full-width without this...

**Review by BoulderBrains — APPROVED** [review]
👏

**Review by ryan-ludwig — APPROVED** [review]

**Pull request merged**